custom_components/entsoe/coordinator.py

```diff
@@ -190,17 +190,21 @@ def today_data_available(self):
         return len(self.get_data_today()) > MIN_HOURS
 
     def _filter_calculated_hourprices(self, data):
+        # rotation = calculations made upon 24hrs today
         if self.calculation_mode == CALCULATION_MODE["rotation"]:
             return {
                 hour: price
                 for hour, price in data.items()
                 if hour >= self.today and hour < self.today + timedelta(days=1)
             }
+        # sliding = calculations made on all data from the current hour and beyond (future data only)
         elif self.calculation_mode == CALCULATION_MODE["sliding"]:
             now = dt.now().replace(minute=0, second=0, microsecond=0)
             return {hour: price for hour, price in data.items() if hour >= now}
+        # rotation = calculations made on all data of today and tomorrow (48 hrs) if we have 72hrs of data
```

Comment:
Did you mean publish here?

Reply:
Yes

Reply:
I cant update these comments as I'm in the middle of adding/changing the calculation modes and running tests. Please ignore these commentary hickups

Reply:
I won't merge a PR with know errors in comments. You should separate your work into branches so you can switch between them.